### PR TITLE
Fetch and display last release changes from GitHub in modal upgrade &…

### PR DESCRIPTION
**Pull request by Ruk33**
… modal auto update downloaded

## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number: https://github.com/lbryio/lbry-desktop/issues/2186

## What is the current behavior?

When a new version of the app is available, the user can see those by checking the releases on Github.

## What is the new behavior?

When a new version of the app is available, the user can see the new changes of the latest release in the app.
The release log is fetched using GitHub's release service: https://docs.github.com/en/rest/reference/repos#releases

## Other information

image

### CODE REVIEW

File: ui/component/lastReleaseChanges/view.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        {seeReleaseNotes}
+      </div>
+    );
+  }
+
+  return (
+    <div>
+      {releaseVersionTitle}
+      <p>
+        {
```
Comment (seanyesmunt) [inline ui/component/lastReleaseChanges/view.jsx:77]: Please try using the `<MarkdownPreview />` component here instead

**Review by seanyesmunt — CHANGES_REQUESTED** [review]
This is awesome! I left one comment which should help with the styling

**Comment by kauffj** [comment]
:heart: :heart: :heart:

File: ui/component/lastReleaseChanges/view.jsx (hunk 1 above)
Comment (Ruk33) [reply ui/component/lastReleaseChanges/view.jsx]: Nice! thanks @seanyesmunt, it's way better now.

**Comment by Ruk33** [comment]
Hello @seanyesmunt , do you think this is good to merge? Please let me know if there is anything else to fix. Thanks! Regards.

**Comment by seanyesmunt** [comment]
@Ruk33 Sorry this has been sitting for so long. I will give this one last test then merge it today!

**Comment by Ruk33** [comment]
Sure @seanyesmunt no worries. Please let me know if you need help with anything. Regards.

**Pull request merged**